DTAdmin/Student/Controllers/EditStudentViewController.swift

```diff
@@ -0,0 +1,201 @@
+        if let selectedImage = info[UIImagePickerControllerOriginalImage] as? UIImage {
+            let size = selectedImage.size
+            let imageHeight: CGFloat = 100.0
+            let aspectRatioForWidth = ( size.width / size.height ) * imageHeight
+            let resizedImage = selectedImage.convert(toSize: CGSize(width: aspectRatioForWidth, height: imageHeight), scale: UIScreen.main.scale)
+            studentPhoto.image = resizedImage
+        } else {
+            showWarningMsg(NSLocalizedString("Image not selected!", comment: "You have to select image to adding in profile."))
+        }
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    func showStudentPhoto(){
+        guard let photoBase64 = studentLoaded?.photo else { return }
+        guard let dataDecoded : Data = Data(base64Encoded: photoBase64, options: .ignoreUnknownCharacters) else { return }
+        let decodedimage = UIImage(data: dataDecoded)
+        studentPhoto.image = decodedimage
+    }
+    
+    @objc func postUpdateStudentToAPI(){
+        if prepareForSave(){
+            guard let userIDForUpdate = studentLoaded?.userId else { return }
+            guard let studentForSave = studentForSave else { return }
+            DataManager.shared.updateEntity(byId: userIDForUpdate, entity: studentForSave, typeEntity: .Student) { error in
+                if let error = error {
+                    self.showWarningMsg(error)
+                } else {
+                    if let resultModification = self.resultModification {
+                        resultModification(studentForSave, false)
+                    }
+                    self.navigationController?.popViewController(animated: true)
+                }
+            }
+        }
+    }
+    
+    @objc func postNewStudentToAPI(){
+        if prepareForSave(){
+            guard let studentForSave = studentForSave else { return }
+            DataManager.shared.insertEntity(entity: studentForSave, typeEntity: .Student) { (id, error) in
+                if let error = error {
+                    self.showWarningMsg(error)
+                } else {
+                    guard let id = id else {
+                        self.showWarningMsg(NSLocalizedString("Incorect response structure", comment: "New user ID not found in the response message"))
+                        return
+                    }
+                    let newUserId = String(describing: id)
+                    var newStudent = studentForSave
+                    newStudent.userId = newUserId
+                    if let resultModification = self.resultModification {
+                        resultModification(newStudent, true)
+                    }
+                    self.navigationController?.popViewController(animated: true)
+                }
+            }
+        }
+    }
+    func prepareForSave() -> Bool {
```

Comment:
It's not clear what does this function do? When it returns true and when false?
Please write Swift doc for it